app/src/main/java/org/ole/planet/myplanet/ui/courses/CoursesFragment.kt

```diff
@@ -79,7 +79,7 @@ class CoursesFragment : BaseRecyclerFragment<RealmMyCourse?>(), OnCourseItemSele
         userModel = UserProfileDbHandler(requireContext()).userModel
         searchTags = ArrayList()
         initializeView()
-        if (isMyCourseLib) {
+        if (isMyCourseLib && !(selectedItems.isNullOrEmpty())) {
```

Comment:
this change does not fix the issue created as this completely hides the leave and archive button